Templating/Helper/ImagineHelper.php

```diff
@@ -40,7 +43,13 @@ public function __construct(CacheManager $cacheManager)
      */
     public function filter($path, $filter, array $runtimeConfig = array())
     {
-        return $this->cacheManager->getBrowserPath($path, $filter, $runtimeConfig);
+        $pathParts = explode('?', $path, 2);
```

Comment:
I'd love to hear a better name than `$pathParts` :roll_eyes:

I tried `list($path, $queryString)`, which looks a lot better, but doesn't work when the original path doesn't contain a question mark (undefined index 1)